src/components/breadcrumb/BreadcrumbNav/BreadcrumbNav.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Breadcrumb } from '../Breadcrumb/Breadcrumb'
+import { BreadcrumbNav } from './BreadcrumbNav'
+
+export default {
+  title: 'Breadcrumb',
+  component: BreadcrumbNav,
+  parameters: {
+    info: `
+    Provide secondary navigation to help users understand where they are in a website.
+
+    Source: https://designsystem.digital.gov/components/breadcrumb/
+    `,
+  },
+}
+
+export const DefaultBreadcrumb = (): React.ReactElement => (
+  <BreadcrumbNav>
+    <Breadcrumb label="Home" href="#" />
+    <Breadcrumb label="Federal Contracting" href="#" />
+    <Breadcrumb label="Contacting assistance programs" href="#" />
+    <Breadcrumb
+      label="Women-owned small business federal contracting program"
+      current
+    />
+  </BreadcrumbNav>
```

Comment:
I'm going to kind of word vomit out a few thoughts on this PR as it stands, starting here!

I like the developer api of this component as used here, and it aligns with the Props Interface requirement defined in the [github issue](https://github.com/trussworks/react-uswds/issues/337#issue-663147524) 

That said, in the case of metaData, there are optional props being passed to Breadcrumb that might make the developer api overly confusing. I would prefer those props to be implementation details. Similar to our FooterNav, for example, we could specify an object structure for the breadcrumb links, and pass those into a BreadcrumbNav rather than relying on `children`. That would require slight changes to the props interface, but I think it would be worth it.

While this approach gives us similar flexibility to the uswds implementation, it also allows the user to specify props on individual breadcrumbs that might be ignored when the BreadcrumbNav tries to enforce metaData on all of its children.